meta-mbl-distro/conf/distro/mbl-linked-repositories.conf

```diff
@@ -24,7 +24,7 @@ SRCREV_MBL_CORE_REPO = "e08ededb896d2324f0b1b20fe6ae5b83417fc6c7"
 SRC_URI_OPTIONS_MBL_CLOUD_CLIENT_INTERNAL_REPO ??= ""
 SRC_URI_MBL_CLOUD_CLIENT_INTERNAL_REPO = "\
     git://git@github.com/ARMmbed/mbl-cloud-client-internal.git;nobranch=1;protocol=ssh;${SRC_URI_OPTIONS_MBL_CLOUD_CLIENT_INTERNAL_REPO};"
-SRCREV_MBL_CLOUD_CLIENT_INTERNAL_REPO = "ed5d641571c25df6f924739a58be64cee9885520"
+SRCREV_MBL_CLOUD_CLIENT_INTERNAL_REPO = "6f1b42d87bab8907111acae6db9102b9662dd0c1"
```

Comment:
You are changing the SHA of the wrong variable.

Reply:
Good catch!